### DISCUSSION TITLE
How to make ebusd load config after issuing "reload"

### DISCUSSION THREAD

**Discussion by jonathanschneider** [john30/ebusd#1281] [Q&A]
Hi all

After changing a type in a config file to i.e. value, I use the `reload` command. The previously loaded config files are then "forgotten". When I `scan` the hardware is detected but no config files are loaded. So far I've only managed to get ebusd to load config files again by restarting the service. Is there another way to get ebusd to load the config files after `reload`?

Thanks,
Jon

**Comment by JonesGW-MGD** [comment]
Are you sure it's not re-loading?(!)  Have you tried `info` rather than  `scan`?

It seems to work fine for me, in that if I do a

`ebusctl reload`

and then a 

`ebusctl info`

I can tell it's reloaded them because doing 'info' instantly shows nothing loaded and repeating a couple of seconds later shows the csv loaded again:

<img width="2309" alt="image" src="https://github.com/john30/ebusd/assets/68991992/3701ff27-99c3-4467-b511-e8e8394cb332">

**Comment by jonathanschneider** [comment]
That's the behaviour I would expect, but no, it's not happening here. Interestingly after your first `info` it doesn't say "scanned" after the addresses as it does for me:

```
localhost: reload
done

localhost: info
version: ebusd 23.3.23.3-5-g47d2a3aa
update check: revision 23.3 available, vaillant/08.bai.csv: different version available, vaillant/15.470.csv: different version available, vaillant/bai.0010003886.inc: different version available
device: ebus-adapter.fritz.box:9999, TCP, enhanced, firmware 1.1[451a].1[451a]
signal: acquired
symbol rate: 21
max symbol rate: 119
min arbitration micros: 1
max arbitration micros: 10
min symbol latency: 6
max symbol latency: 29
scan: finished
reconnects: 0
masters: 1
messages: 11
conditional: 0
poll: 0
update: 4
address 03: master #11
address 08: slave #11, scanned
address 10: master #2
address 15: slave #2, scanned
address 31: master #8, ebusd
address 36: slave #8, ebusd
```
